### PR TITLE
Php8 support

### PR DESCRIPTION
**Pull request by wbloszyk**
## Subject

I am targeting this branch, because this is BC. 

## Changelog

```markdown
### Added
- Added support for PHP 8.0
```

### CODE REVIEW

**Comment by VincentLanglet** [comment]
PR is needed on webkit

**Comment by wbloszyk** [comment]
> PR is needed on webkit

https://github.com/sonata-project/dev-kit/pull/1688

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -27,6 +27,7 @@ jobs:
                 php-version:
                     - '7.3'
                     - '7.4'
+                    - '8.0'
```
Comment (VincentLanglet) [inline .github/workflows/test.yaml:30]: You allow need to remove the part 
```
name: Configuration required for PHP 8.0
```
to be sure it's compatible with php8

Comment (wbloszyk) [reply .github/workflows/test.yaml:30]: I missed it. Done.

**Review by VincentLanglet — APPROVED** [review]

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**